src/components/SiteFormComponent.vue

```diff
@@ -1,105 +1,183 @@
+  city: '',
+  zip: '',
+  province: '',
+  country: '',
+});
+
+interface Tenant {
+  id: number;
+  firstName: string;
+  lastName: string;
+  period: string;
+  price: number;
+  deposit: number;
+  extraCosts: number;
+  email: string;
+}
+
+const items = ref<Tenant[]>([
   {
-    name: 'zip',
-    label: 'PLZ',
-    type: 'text',
-    required: false,
+    id: 1,
+    firstName: 'Lena',
+    lastName: 'Schneider',
+    period: '01.06.2025 - 11.06.2025',
+    price: 125,
+    deposit: 10,
+    extraCosts: 13,
+    email: 'lena.schneider@example.com',
   },
+]);
+
+const formerItems = ref<Tenant[]>([
   {
-    name: 'province',
-    label: 'Bundesland',
-    type: 'text',
-    required: false,
+    id: 2,
+    firstName: 'Tobias',
+    lastName: 'Keller',
+    period: '01.01.2023 - 31.12.2024',
+    price: 120,
+    deposit: 10,
+    extraCosts: 12,
+    email: 'tobias.keller@example.com',
   },
   {
-    name: 'country',
-    label: 'Land',
-    type: 'text',
-    required: false,
+    id: 3,
+    firstName: 'Miriam',
+    lastName: 'Fischer',
+    period: '01.01.2021 - 31.12.2023',
+    price: 115,
+    deposit: 9,
+    extraCosts: 12,
+    email: 'miriam.fischer@example.com',
   },
-];
+]);
 
-let initialValues = ref(props.initialValues);
+const showFormer = ref(false);
+// Backend-Aufruf auskommentiert
+/*
+onMounted(async () => {
+  const url = `/projects/${props.projectId}/sites/${props.siteId}/tenancies`;
+  try {
+    const res = await fetch(url);
+    const data = await res.json();
+    items.value = data.current || [];
+    formerItems.value = data.former || [];
+  } catch {
+    items.value = [];
+    formerItems.value = [];
+  }
+});
+*/
 
-if (!props.initialValues) {
-  initialValues = ref({
-    title: '',
-    description: '',
-    usableSpace: '',
-    street: '',
-    city: '',
-    zip: '',
-    province: '',
-    country: '',
-  });
-}
 </script>
 
 <template>
   <ReusableForm
-    :fields="fields"
-    :initialValues="initialValues"
-    :headline="headline"
-    :saveButtonText="saveButtonText"
-    :cancelButtonText="cancelButtonText"
-    :onSubmit="onSubmit"
-    :onCancel="onCancel"
+      :fields="fields"
+      :initialValues="initialValues"
+      :headline="headline"
+      :saveButtonText="saveButtonText"
+      :cancelButtonText="cancelButtonText"
+      :onSubmit="onSubmit"
+      :onCancel="onCancel"
   />
+  <div class="p-6 max-w-4xl mx-auto mt-10 shadow-lg bg-white rounded">
+    <h2 class="text-xl font-bold mb-4">Aktuelle Mieter</h2>
+    <DataTable :value="items" class="w-full mb-4">
+      <Column field="id" header="ID" />
+      <Column field="firstName" header="Vorname" />
+      <Column field="lastName" header="Nachname" />
+      <Column field="period" header="Zeitraum" />
+      <Column field="price" header="Preis" />
+      <Column field="deposit" header="Anzahlung" />
+      <Column field="extraCosts" header="Extra Kosten" />
+      <Column field="email" header="E-Mail" />
+    </DataTable>
```

Comment:
In der `DataTable` für Mieter könnte ein `#empty`-Slot ergänzt werden, um bei leerer Liste z. B. den Hinweis „Keine Mieter vorhanden.“ anzuzeigen.

also sowas wie

```suggestion
  <template #empty>
      <div class="text-center text-gray-500 py-4">
         Keine Mieter vorhanden.
       </div>
      </template>
    </DataTable>
```